support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.jsx

```diff
@@ -77,7 +77,11 @@ const weeklyProductProps = (billingPeriod: WeeklyBillingPeriod, productPrice: Pr
     buttonCopy: 'Subscribe now',
     href: getCheckoutUrl(billingPeriod, orderIsAGift),
     label: getPromotionLabel(promotion) || '',
-    onClick: sendTrackingEventsOnClick(`subscribe_now_cta-${billingPeriod}`, 'GuardianWeekly', null),
+    onClick: sendTrackingEventsOnClick({
+      id: `subscribe_now_cta-${billingPeriod}`,
```

Comment:
Updated the id to include `billingPeriod` so we can get more info on the CTA selected.

Reply:
Corresponding Trello Card [here](https://trello.com/c/ynLFxBjM/3567-update-click-tracking-component-id-on-pricing-cards-on-the-digital-subs-paper-product-landing-pages-to-illustrate-period).